CHANGELOG.md

```diff
@@ -39,6 +39,12 @@ All notable changes to this project will be documented in this file.
 * ### NI-SCOPE
     * #### Added
     * #### Changed
+        * `niscope.fetch()` and `niscope.fetch_into()` now takes additional parameters for associated attributes and add resulting record as part of the waveform info
```

Comment:
Should be `niscope.Session.fetch()` or since you are inside the NI-SCOPE section `Session.fetch()`

Reply:
Changed